packages/react/src/drafts/IssueLabel/IssueLabel.tsx

```diff
@@ -43,8 +45,22 @@ export interface IssueLabelProps extends React.PropsWithChildren {
 }
 
 export function IssueLabel({children, fillColor, size = 'small', variant = 'gray', ...rest}: IssueLabelProps) {
+  const {resolvedColorScheme} = useTheme()
+  const mode = resolvedColorScheme?.startsWith('dark') ? 'dark' : 'light'
+  // I define those bgColors as we can't easily get them from primitives,
+  // they are needed to calc contrast in getColorsFromHex()
+  const bgColors = {
+    light: '#ffffff',
+    dark: '#0d1117',
+  }
```

Comment:
I think with the hex color change that this will need to be updated to be something like:

```suggestion
  const bgColors: Record<string, Hex> = {
    light: '#ffffff',
    dark: '#0d1117',
  }
```

So that when we index into it below on line 62 it will infer it as a hex color